src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -95,6 +95,19 @@ internal abstract class BrokerClient : IBrokerClient {
             }
         }
 
+        public async Task<bool> DeleteRemoteUserProfileAsync(CancellationToken cancellationToken) {
+            try {
+                if (HttpClient != null) {
+                    var response = await HttpClient.DeleteAsync("deleteuser", cancellationToken);
```

Comment:
There should be a service class for it, similar to `SessionsWebService`. This endpoint requires authentication, so you want the code that is there to deal with it to be involved. It will also support `BrokerApiError` correctly, and throw the corresponding exception, which can then be caught here (like in `ConnectAsync` below).